app/(storyblok)/[[...slug]]/page.tsx

```diff
@@ -22,12 +21,35 @@ type ParamsType = {
 // Control what happens when a dynamic segment is visited that was not generated with generateStaticParams.
 export const dynamicParams = false;
 
+// Storyblok bridge options.
+const bridgeOptions = {
+  preventClicks: true,
+  resolveLinks: 'story',
+};
```

Comment:
Why these are an option of the StoryblokStory component has got me.

https://github.com/storyblok/storyblok-react/blob/main/lib/story.tsx